python/setup.py

```diff
@@ -174,6 +174,7 @@ def find_version(*filepath):
     "six >= 1.0.0",
     "faulthandler;python_version<'3.3'",
     "protobuf >= 3.8.0",
+    "psutil",
```

Comment:
Unfortunately, we can't depend with psutil since it doesn't ship with binaries. This would break ray installs for any system without gcc present.

Reply:
😢